tests/test_syntax.py

```diff
@@ -46,6 +46,42 @@ def test_python_render():
     assert rendered_syntax == expected
 
 
+def test_get_theme():
+    syntax = Syntax(
+        CODE,
+        lexer_name="python",
+        line_numbers=True,
+        line_range=(2, 10),
+        theme="foo",
+        code_width=60,
+        word_wrap=True,
+    )
+    from pygments.style import Style as PygmentsStyle
+
+    style = PygmentsStyle()
+    assert syntax.get_theme(style).get_background_style().bgcolor.name == "#ffffff"
+
+
+def test_pygments_syntax_theme():
+    style = PygmentsSyntaxTheme("default")
+    assert style.get_style_for_token("abc") == Style.parse("none")
+
+
+def test_highlight_background_color():
+    syntax = Syntax(
+        CODE,
+        lexer_name="python",
+        line_numbers=True,
+        line_range=(2, 10),
+        theme="foo",
+        code_width=60,
+        word_wrap=True,
+        background_color="red",
+    )
+    text = syntax.highlight(CODE)
+    assert syntax.highlight(CODE) == text
```

Comment:
This test is guaranteed to pass. So not much point in having it.

Reply:
yeah, will update the test